lib/ansible/modules/windows/win_file.ps1

```diff
@@ -52,29 +41,29 @@ namespace Ansible.Command {
 Add-Type -TypeDefinition $symlink_util
 
 # Used to delete directories and files with logic on handling symbolic links
-function Remove-File($file, $checkmode) {
+function Remove-File($File, $WhatIf) {
```

Comment:
Why change to pascal case, we already have a set way to writing modules in here and would prefer to keep it consistent.

Reply:
I didn't know we had a standard already. The reason I renamed it to -WhatIf is because it's more clear this is directly affecting lower level Powershell cmdlets. IMO that's more clear.

Reply:
There isn't a global standard but we try and keep each script aligned to how they are written. In this case it seems like a semi snake case is being used and it would be preferable to keep that standard. Personally I find it nice that our custom functions are lower case as it sets them apart from the standard MS cmdlets but in the end as long as it is consistent I am fine with it.

Reply:
Jordan, that's assuming it was consistent.

The original module was also inconsistent in this regard and I had two options to make it more consistent, I chose CamelCase (as that's was used for internal functions as well in the main loop).

But have it your way :-) I don't like CamelCase either, and I hate case insensitive language with a vengeance ! But now it looks more like potpourri than before IMO.

Reply:
I don't think it was inconsistent before, having a look all the variables are in `snake_case` format. I agree we do things differently when calling our functions compared to the inbuilt functions like `-file` vs `-File` but all the variables we set were in a camel case style and I think it should be consistent in the style the module is currently in.

I'll review the changes tomorrow when I wake up and start work.

Reply:
The module did call internal functions with `-CheckMode` and `-File`, as well as `-checkmode` and `-file`. So there were 2 ways to go with that. I chose CamelCase to make everything CamelCase.

So my original proposal was more consistent, both with the original and the current proposed PR according to your liking. I don't care, but the rules become very blurry if 

1. the file itself is inconsistent
2. the original file style should be maintained
3. it ends up being someone's preference anyhow

End I have to rewrite my PR which is just a waste of my time. Just saying...